include/chemfiles/files/XTCFile.hpp

```diff
@@ -0,0 +1,47 @@
+// Chemfiles, a modern library for chemistry file reading and writing
+// Copyright (C) Guillaume Fraux and contributors -- BSD license
+
+#ifndef CHEMFILES_XTC_FILE_HPP
+#define CHEMFILES_XTC_FILE_HPP
+
+#include "chemfiles/File.hpp"
+#include "xdrfile.h"
+#include "xdrfile_xtc.h"
+
+namespace chemfiles {
+
+/// Simple RAII capsule for `XDRFILE*`, handling the creation and
+/// destruction of the file as needed.
```

Comment:
This could also mention that the class reads the file header to get frames offset (either here or in the constructor documentation)

Reply:
Done.